### PR TITLE
Add setContentSize support

### PR DESCRIPTION
**Pull request by maddie**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -170,7 +170,7 @@ function onReady () {
                 client.write(json.targetID, consts.eventNames.sessionEventLoadedExtension)
             })
             break;
-                
+
```
Comment (asticode) [inline index.js:173]: can you remove this change from the PR ?

File: index.js [hunk 2]
```diff
@@ -453,6 +456,7 @@ function windowCreateFinish(json) {
     elements[json.targetID].on('move', () => { client.write(json.targetID, consts.eventNames.windowEventMove) })
     elements[json.targetID].on('ready-to-show', () => { client.write(json.targetID, consts.eventNames.windowEventReadyToShow) })
     elements[json.targetID].on('resize', () => { client.write(json.targetID, consts.eventNames.windowEventResize) })
+    elements[json.targetID].on('resize-content', () => { client.write(json.targetID, consts.eventNames.windowEventResizeContent) })
```
Comment (asticode) [inline index.js:459]: have you tested this is working properly ?

because you forgot to add `windowEventResizeContent` in the constants

**Review by asticode — CHANGES_REQUESTED** [review]

File: index.js (hunk 2 above)
Comment (maddie) [reply index.js:459]: You're right, I added in my local repo but the `git push -f` didn't go through. Should be fixed now!

File: index.js (hunk 1 above)
Comment (maddie) [reply index.js:173]: Ahhh sorry, didn't see this comment here. I think this is because of `prettier` auto-formatting on save.

Comment (asticode) [reply index.js:173]: no worries, those blanks were actually useless 👍

**Pull request merged**